packages/storybook/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@nrwl/storybook",
+  "version": "0.0.1",
+  "description": "Storybook plugin for Nx",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nrwl/nx.git"
+  },
+  "keywords": [
+    "Monorepo",
+    "Angular",
+    "React",
+    "Web",
+    "Node",
+    "Nest",
+    "Jest",
+    "Storybook",
+    "Cypress",
+    "CLI"
+  ],
+  "main": "index.js",
+  "types": "index.d.ts",
+  "author": "Isaac Mann",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "homepage": "https://nx.dev",
+  "schematics": "./collection.json",
+  "builders": "./builders.json",
+  "ng-update": {
+    "requirements": {},
+    "migrations": "./migrations.json"
+  },
+  "peerDependencies": {
+    "@nrwl/workspace": "*",
+    "@storybook/angular": "^5.1.9",
+    "@storybook/addon-knobs": "^5.1.9",
+    "@types/storybook__addon-knobs": "^5.1.9",
```

Comment:
fyi https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38447#issuecomment-541112865 a list of deleted types from @types that are included in storybook 5.2.x